imod/constants.py

```diff
@@ -26,18 +26,25 @@
 # *
 # *****************************************************************************
 
-
+# IMOD
 IMOD_HOME = 'IMOD_HOME'
 IMOD_VIEWER_BINNING = 'IMOD_VIEWER_BINNING'
 ETOMO_CMD = 'etomo'
-
 VERSION_4_11_24 = '4.11.24'
```

Comment:
Just a comment, IMOD 5.1 has been released, we should check. At least the CTF was released